addons/cargo/functions/fnc_unloadItem.sqf

```diff
@@ -20,10 +21,10 @@
  * Public: Yes
  */
 
-params [["_item", "", [objNull, ""]], ["_vehicle", objNull, [objNull]], ["_unloader", objNull, [objNull]], ["_deploy", []]];
+params [["_item", "", [objNull, ""]],["_vehicle", objNull, [objNull]],["_unloader", objNull, [objNull]],["_deploy", []],["_checkVehicleIsStable", true, [true]]];
```

Comment:
Either add the space between the arguments back or separate them on multiple lines, like this:
```suggestion
params [
    ["_item", "", [objNull, ""]],
    ["_vehicle", objNull, [objNull]],
    ["_unloader", objNull, [objNull]],
    ["_deploy", []],
    ["_checkVehicleIsStable", true, [true]]
];
```